src/coreclr/jit/compiler.h

```diff
@@ -8318,6 +8318,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     SIMDLevel getSIMDSupportLevel()
```

Comment:
Not something you need to handle in this PR, most just a general comment `getSIMDSupportLevel` is effectively "dead" at this point. It's probably something we should remove.

There is one asserting in `codegenxarch.cpp` that should just be checking `IsBaselineSimdIsaSupportedDebugOnly` instead and the couple usages in `compiler.h` below should be replaced with direct checks against `compOpportunisticallyDependsOn` instead.